lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb

```diff
@@ -2135,7 +2139,7 @@ def air_loop_hvac_adjust_minimum_vav_damper_positions(air_loop_hvac)
       v_dz = v_pz * mdp
 
       # Zone discharge air fraction
-      z_d = v_oz / v_dz
+      z_d = (v_dz.zero? || v_oz.zero?) ? 0.0 : v_oz / v_dz
```

Comment:
`v_pz` shows up in the denominator again below in line 2164, which should also guard against `v_pz.zero?`

Reply:
added a guard clause to line 2164